software/digistorm.yml

```diff
@@ -0,0 +1,11 @@
+name: "Digistorm"
+website_url: "https://ladigitale.dev/"
```

Comment:
```suggestion
website_url: "https://digistorm.app/"
```

Let's use the main instance as `website_url`, since it is available in English and correctly describes what the project does.

Reply:
sounds good, will do that for future ones!